pkg/server/ca/ca_test.go

```diff
@@ -424,25 +424,61 @@ func (s *CATestSuite) createJWTSVIDParams(trustDomain spiffeid.TrustDomain, ttl
 }
 
 func (s *CATestSuite) createCACertificate(cn string, parent *x509.Certificate) *x509.Certificate {
+	return createCACertificate(s.T(), s.clock, cn, parent)
+}
+
+func TestOmitX509SVIDUID(t *testing.T) {
+	clk := clock.NewMock(t)
+	log, _ := test.NewNullLogger()
+
+	ca := NewCA(Config{
+		Log:         log,
+		Metrics:     telemetry.Blackhole{},
+		TrustDomain: trustDomainExample,
+		X509SVIDTTL: time.Minute,
+		Clock:       clk,
+		CASubject: pkix.Name{
+			CommonName: "TESTCA",
```

Comment:
this can be a little confusing to me at least, 
we are validating that subject is not used, but this subject does not provide country or organization that are the ones we set from CA into an SVID,
Can you add Country or Organization here so we can be sure we are not setting values from there?

Reply:
No longer relevant since we've removed CA subject inheritance.